src/main/menus/tray.js

```diff
@@ -29,6 +29,14 @@ function createTemplate(mainWindow, config) {
     }), {
       type: 'separator'
     }, {
+      label: 'Settings',
+      click: () => {
+        mainWindow.loadURL('file://' + __dirname + '/browser/settings.html');
+        mainWindow.show();
```

Comment:
On Linux Mint 17, it is working. Even if you're not on the good workspace, it switches your ws and open up the window.
Is it something proper to OSX/Windows ?